client/src/helpers/api.js

```diff
@@ -1,7 +1,8 @@
+import axios from "axios";
 const API_URL = process.env.REACT_APP_API_URI || "http://localhost:4000";
 
 export const getStatus = () => {
-  return fetch(`${API_URL}/api/status`).then(res => res.json());
+  return axios.get(`${API_URL}/api/status`).then(res => res.json());
```

Comment:
since we're using `axios`, we can use baseUrl property in config instead of concatenating strings.